router/client/src/client.rs

```diff
@@ -121,11 +121,17 @@ impl Client {
         };
 
         // get all possible batch sizes
-        let decode_bucket_size: u32 = read_env_var("BATCH_BUCKET_SIZE", 8);
+        let mut decode_bucket_size: u32 = read_env_var("BATCH_BUCKET_SIZE", 8);
         let max_decode_batch_size: u32 = match max_batch_size {
             Some(max_batch_size) => max_batch_size as u32,
             None => decode_bucket_size
         };
+
+        let max_total_tokens_2_bs_pair_str = std::env::var("MAX_TOTAL_TOKENS_2_MAX_BATCH_SIZE_LIST".to_string()).unwrap_or("".to_string()).to_string();
```

Comment:
This change seems a bit overcomplicated. Perhaps it could be expressed as:

let decode_bucket_size = match(std::env::var("MAX_TOTAL_TOKENS_2_MAX_BATCH_SIZE_LIST")) {
    Ok(key) => max_decode_batch_size,
   Err(_) = > decode_bucket_size,
}

Reply:
Good suggestion! Will update later. Thanks！

Reply:
Updated.